Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -99,13 +99,89 @@ class KernFeatureWriter(BaseFeatureWriter):
     """Generates a kerning feature based on groups and rules contained
     in an UFO's kerning data.
 
-    There are currently two possible writing modes:
-    2) "skip" (default) will not write anything if the features are already present;
-    1) "append" will add additional lookups to an existing feature, if present,
-       or it will add a new one at the end of all features.
+    There are currently two possible writing modes: 2) "skip" (default) will not
```

Comment:
since you're touching this docstring, you may mention the fact that in the default "skip" mode the insertion marker comments are being taken into account and applied if present (unlike in "append" mode)

Reply:
I'll remove it and point to the base writer docstring.